app/trip_cost.py

```diff
@@ -0,0 +1,17 @@
+from app.customer import Customer
+from app.shop import Shop
+
+
+def calculate_trip_cost(
+        customer: Customer,
+        shop: Shop,
+        fuel_price: float
+) -> float:
+    distance = (abs(customer.location[0] - shop.location[0])
+                ** 2 + abs(customer.location[1] - shop.location[1])
+                ** 2) ** 0.5

```

Comment:
Ensure that `customer.location` and `shop.location` are tuples or lists with at least two elements. If not, accessing these indices will raise an `IndexError`.